.github/config.yml

```diff
@@ -17,10 +17,9 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshery-consul/master/.github/welcome/Layer5-celebration.png)
+  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshery/master/.github/welcome/Layer5-celebration.png)
   
-  :star: Please [star the project](../stargazers) if you have yet to do so.
-
+                          :star: Please [star the project](../stargazers) if you have yet to do so.
```

Comment:
Why so many spaces are added? 😕

Reply:
Good question, @JubayerJoy. Normally, such whitespace is not warranted (or even honored depending the system interpreting it). This might be an issue I caused. I tossed in some similar extra whitespace earlier to into this (or another) `config.yml`, because this message was getting visually lost under the celebration picture. I want to ensure that people see the call for stars. We need them! 😄 🤩

Reply:
@leecalcote Oh! I get it 💯